test/modules/aws-codebuild-integration.ts

```diff
@@ -202,6 +202,35 @@ phases:
   `),
   );
 
+  it(
+    'adds a new codebuild_project with codepipeline type',
+    query(`
+    INSERT INTO codebuild_project (project_name, source_type, service_role_name)
```

Comment:
if codepipeline is not installed and code-build doesn't depend on it, how is this working?

Reply:
because it does nothing with it... it just creates the project and sits there. The build is not started until i add the project on the pipeline itself